site/docs/themes/index.mdx

```diff
@@ -12,7 +12,7 @@ Salt theme is a collection of defined visual styles that shape the overall look-
 
 A theme in Salt enables delivery teams to streamline the process of creating a consistent design language across their business domain, platforms or products.
 
-### Design tokens
+### Design Tokens
```

Comment:
Sentence case for heading is part of content guide

Reply:
"Design Tokens" is title case, sentence case is where only the first letter of the sentence is capitalised.